requirements.txt

```diff
@@ -30,8 +30,8 @@ sentry-sdk~=2.10.0
 shapely~=2.0.1
 simplejson~=3.19.1
 stripe~=7.10.0
-tensorflow~=2.17.0
+tensorflow
```

Comment:
@lukemadera curious if you have another approach, but I was getting this error when this was defined:

> ERROR: Ignored the following versions that require a different python version: 1.21.2 Requires-Python >=3.7,<3.11; 1.21.3 Requires-Python >=3.7,<3.11; 1.21.4 Requires-Python >=3.7,<3.11; 1.21.5 Requires-Python >=3.7,<3.11; 1.21.6 Requires-Python >=3.7,<3.11
> ERROR: Could not find a version that satisfies the requirement tensorflow~=2.17.0 (from versions: 2.12.0rc0, 2.12.0rc1, 2.12.0, 2.12.1, 2.13.0rc0, 2.13.0rc1, 2.13.0rc2, 2.13.0, 2.13.1, 2.14.0rc0, 2.14.0rc1, 2.14.0, 2.14.1, 2.15.0rc0, 2.15.0rc1, 2.15.0, 2.15.1, 2.16.0rc0, 2.16.1, 2.16.2)
> ERROR: No matching distribution found for tensorflow~=2.17.0

my pip freeze looks like this now (I think because it's a mac it has no tensorflow module):

> tensorboard==2.12.3
> tensorboard-data-server==0.7.2
> tensorflow-estimator==2.12.0
> tensorflow-io-gcs-filesystem==0.37.1
> tensorflow-macos==2.12.0
> tensorflow-metal==1.0.0

Reply:
Can you try `pip uninstall tensorflow-macos tensorflow-metal`? Then try again? That's what I had to do and what worked for me. Are you on python v3.11? I think we want to set a version, otherwise we'll run into other errors in the future.

Reply:
still getting this error
> ERROR: Could not find a version that satisfies the requirement tensorflow~=2.17.0 (from versions: 2.12.0rc0, 2.12.0rc1, 2.12.0, 2.12.1, 2.13.0rc0, 2.13.0rc1, 2.13.0rc2, 2.13.0, 2.13.1, 2.14.0rc0, 2.14.0rc1, 2.14.0, 2.14.1, 2.15.0rc0, 2.15.0rc1, 2.15.0, 2.15.1, 2.16.0rc0, 2.16.1, 2.16.2)
ERROR: No matching distribution found for tensorflow~=2.17.0

but it installs 2.16.2 when I give it the choice. is 2.17.0 the version you are currently on?

also, I am available to screen share if needed today until 1pm ET/10am PT or tomorrow 10 ET/7 PT or 11 ET/8 PT. let me know if you have a thought on the above or if you have time to screen share today or tomorrow.

Reply:
That is so odd. What is your `pip list`?
Here is mine:
```
tensorboard                  2.17.1
tensorboard-data-server      0.7.2
tensorflow                   2.17.0
tensorflow-io-gcs-filesystem 0.36.0
```

Reply:
ok, just upgraded pip to 24.2 (I was on 22.3), still having the same issue though. Mac version is Monterey 12.7.6 intel based. I was wondering if maybe it's related to my Mac/Xcode version, but they are the highest this laptop can go.

when I keep the tensorflow version out of my requirements, and pip install runs completely, this is my pip list (otherwise tensorflow won't install):

> tensorboard                  2.16.2
tensorboard-data-server      0.7.2
tensorflow                   2.16.2
tensorflow-io-gcs-filesystem 0.37.1

I'm thinking maybe for now I can just run on 2.16.2 and just remove the requirements change, and see if I run into any issues. What do you think?